Makefile

```diff
@@ -41,30 +41,25 @@ GCFLAGS="all=-N -l"
 ENVTEST_K8S_VERSION = 1.28.0
 
 TEST_MANIFEST_DIR_DEFAULT ?= tests/manifests/default
-TEST_MANIFEST_DIR_E2E ?= tests/manifests/e2e
-TEST_NOSTRATEGY_MANIFEST_DIR ?= tests/manifests/special-cases/no-strategy
-TEST_PROGRESSIVE_MANIFEST_DIR ?= tests/manifests/special-cases/progressive
+TEST_MANIFEST_DIR_E2E_DEFAULT ?= tests/manifests/e2e/default
+TEST_NOSTRATEGY_MANIFEST_DIR ?= tests/manifests/e2e/special-cases/no-strategy
+TEST_PAUSE_AND_DRAIN_MANIFEST_DIR ?= tests/manifests/e2e/special-cases/pause-and-drain
 
 TEST_MANIFEST_DIR := $(TEST_NOSTRATEGY_MANIFEST_DIR)
 
-ifeq ($(TEST_TYPE), e2e)
-TEST_MANIFEST_DIR := $(TEST_MANIFEST_DIR_E2E)
+ifeq ($(STRATEGY), progressive)
+TEST_MANIFEST_DIR := $(TEST_MANIFEST_DIR_E2E_DEFAULT)
 endif
```

Comment:
sorry, just thinking that if `progressive` is truly the default case, then running `make start` should actually do progressive. Maybe we should restructure this as:

```
TEST_MANIFEST_DIR := $(TEST_MANIFEST_DIR_E2E_DEFAULT)

ifeq ($(STRATEGY), no-strategy)
TEST_MANIFEST_DIR := $(TEST_NOSTRATEGY_MANIFEST_DIR)
endif
```

Reply:
(apologies for the scope creep on my part) :)